pnpm lint fails on a fresh pull

I wanted to see if I can make this work with Voi as well so I pulled it down to tink with it. I don't have much experience working with pnpm or monorepos so I was going to learn along the way. First thing I wanted to do was corepack automatically figure out the package manager and install it and all it would take was adding a single line into the package.json. I made the change and did a lint for the fun of it and it came back with an error:

`packages/react lint$ eslint "**/*.{js,jsx,ts,tsx}"
│ Oops! Something went wrong! :(
│ ESLint: 9.24.0
│ Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'typescript-eslint' imported from /Users/cswenor/Dev/use-wallet-ui/packages/react/eslint.config.js
│ Did you mean to import "typescript-eslint/dist/index.js"?
│     at Object.getPackageJSONURL (node:internal/modules/package_json_reader:268:9)
│     at packageResolve (node:internal/modules/esm/resolve:768:81)
│     at moduleResolve (node:internal/modules/esm/resolve:854:18)
│     at defaultResolve (node:internal/modules/esm/resolve:984:11)
│     at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:685:12)
│     at #cachedDefaultResolve (node:internal/modules/esm/loader:634:25)
│     at ModuleLoader.resolve (node:internal/modules/esm/loader:617:38)
│     at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:273:38)
│     at ModuleJob._link (node:internal/modules/esm/module_job:135:49)
└─ Failed in 269ms at /Users/cswenor/Dev/use-wallet-ui/packages/react
/Users/cswenor/Dev/use-wallet-ui/packages/react:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @txnlab/use-wallet-ui-react@0.2.2 lint: `eslint "**/*.{js,jsx,ts,tsx}"`
Exit status 2
 ELIFECYCLE  Command failed with exit code 2.
`

The fix is pretty simple and it is installing typescript-eslint to the root, but you have both the eslint-plugin and parser in there. I removed them too but not sure if they are needed anywhere else, but because I'm new to this style just wanted to drop an issue here, and I will follow up with the small code changes to make it work.

--- a/package.json
+++ b/package.json
@@ -35,6 +35,7 @@
     "node": ">=22",
     "pnpm": ">=9"
   },
+  "packageManager": "pnpm@9.0.0",
   "devDependencies": {
     "@eslint/js": "^9.21.0",
     "@tanstack/config": "^0.18.0",
@@ -50,7 +51,8 @@
     "eslint-plugin-unicorn": "^57.0.0",
     "globals": "^15.15.0",
     "prettier": "^3.5.3",
-    "typescript": "^5.8.2"
+    "typescript": "^5.8.2",
+    "typescript-eslint": "^8.32.0"
   },
   "pnpm": {
     "overrides": {

